src/main/scala/org/constellation/API.scala

```diff
@@ -124,7 +124,7 @@ class API()(implicit system: ActorSystem, val timeout: Timeout, val dao: DAO)
               complete(ChannelUIOutput(dao.threadSafeMessageMemPool.activeChannels.keys.toSeq))
             } ~
               path("channel" / Segment / "info") { channelId =>
-                complete(dao.channelService.get(channelId).map { cmd =>
+                complete(dao.channelService.getSync(channelId).map { cmd =>
```

Comment:
Why makes all these sync? akka-http can handle async bodies

Reply:
Though I guess we talked about this -- and it's just swaydb which is super fast. so not a big deal

Reply:
I've changed names only. Originally there was `get` only which was synchronous. I've changed `get` to async but added `getSync` to maintain backward compatibility without changing whole codebase (but incrementally during working on sth else)